includes/class-wcs-core-autoloader.php

```diff
@@ -175,12 +175,14 @@ protected function is_class_interface( $class ) {
 	 */
 	protected function is_class_data_store( $class ) {
 		static $data_stores = array(
-			'wcs_related_order_store_cached_cpt'  => true,
-			'wcs_related_order_store_cpt'         => true,
-			'wcs_customer_store_cached_cpt'       => true,
-			'wcs_customer_store_cpt'              => true,
-			'wcs_product_variable_data_store_cpt' => true,
-			'wcs_subscription_data_store_cpt'     => true,
+			'wcs_orders_table_subscription_data_store' => true,
+			'wcs_orders_table_data_store_controller'   => true,
```

Comment:
Yeah that's correct, it's not a data store class but I put this class in `includes/data-store/` directory as it's directly related. Since we had datastore related array in the autoloader i put the controller class in there.